conans/test/integration/cache/backup_sources_test.py

```diff
@@ -185,6 +185,15 @@ def source(self):
         assert f"Sources for {self.file_server.fake_url}/internet/myfile.txt found in remote backup" in self.client.out
         assert f"File {self.file_server.fake_url}/mycompanystorage/mycompanyfile.txt not found in {self.file_server.fake_url}/backups/" in self.client.out
 
+        # Ensure defaults backup folder works if nothing set in global.conf
```

Comment:
Not sure if I understand the comment, you say nothing in the ``global.conf``, then define the ``global.conf``? 😅